glusterapi/georep.py

```diff
@@ -1,58 +1,176 @@
-from common import BaseAPI
+import httplib
+import json
+import socket
+import requests
+
+from glusterapi.common import BaseAPI, validate_volume_name
+from glusterapi.exceptions import GlusterApiError
+from glusterapi.exceptions import GlusterApiInvalidInputs
+
+
+def extract_volume_ID(remotehost, remoteport, remote_endpoint, remotevol, headers):
+    if int(remoteport) not in range(1, 65535):
+        raise GlusterApiInvalidInputs("Incorrect port")
+    try:
+        socket.inet_aton(remotehost)
```

Comment:
`socket.inet_aton` does the only IPV4 validation, what happens when `remotehost` is IPV6 address or DHCP address